string command
uint16 state
---
string response